src/content/plugins/progress-plugin.md

```diff
@@ -0,0 +1,96 @@
+---
+title: ProgressPlugin
+contributors:
+  - elliottsj
+---
+
+The `ProgressPlugin` provides a way to customize how progress is reported during a compilation.
+
+## Usage
+
+Create an instance of `ProgressPlugin` with a handler function which will be called when hooks report progress:
```

Comment:
This is a showcase within your own plugin right? maybe provide a context e.g.

__myPlugin.js__

```js
const handler = (percentage, message, ...args) => {
.....
```

Reply:
It can used inside of a plugin (e.g. [progress-bar-webpack-plugin](https://github.com/clessg/progress-bar-webpack-plugin/blob/cf1d3105aa9b3178c1bca7b021cfc03b5d371e65/index.js#L45)), but ProgressPlugin can also be used standalone, e.g.

```js
// webpack.config.js

module.exports = {
  plugins: [
    new ProgressPlugin((percentage, message, ...args) => {
      console.info(percentage, message, ...args);
    })
  ]
};
```

I agree that it would be useful to showcase / link to examples of using ProgressPlugin within a parent plugin. What if I add an "Examples" section and link to ProgressPlugin usage within progress-bar-webpack-plugin / webpack-cli / webpack-command?

Reply:
the example you gave is much better in my opinion, maybe replace it?

Reply:
Updated to include the `console.info()` line.

Reply:
Do you think I should include the `module.exports = { plugins: [` part too? Leaving it out for now for consistency with the other plugin docs